docker-compose.yaml

```diff
@@ -0,0 +1,22 @@
+version: '3.6'
+services:
+    broker:
+        build:
+            context: .
+            dockerfile: Dockerfile
+        restart: unless-stopped
+        links: 
+            - mongodb
+        command: bash -c "cd /app/broker; python app.py"
+        volumes:
+            - /home/akash/Desktop/ELIXIR/containerMount/broker/:/app
```

Comment:
II have seen your home directory paths at multiple places, can this be replaced with a generic path? Or will the user setting it up has to change it manually?

Reply:
No more needed! I've removed it.